encodings/bytebool/src/compute.rs

```diff
@@ -157,4 +137,14 @@ mod tests {
             ByteBoolArray::from(vec![Some(true), Some(true), None, Some(false), None]).as_ref(),
         );
     }
+
+    #[test]
+    fn test_take_byte_bool() {
+        test_take_conformance(ByteBoolArray::from(vec![true, false, true, true, false]).as_ref());
```

Comment:
Can you generate these as separate tests instead of statements in one test? You can tell it to follow what's done for search_sorted